### PR TITLE
fix(charts/daemonset-app): Remove default probes

### PR DESCRIPTION
**Pull request by scohen-nd**
Remove default probes to fix an issue of multiple probes.

### CODE REVIEW

File: charts/daemonset-app/values.yaml [hunk 1]
```diff
@@ -98,23 +98,17 @@ volumes: []
 # -- A PodSpec container "startupProbe" configuration object. Note that this
 # startupProbe will be applied to the proxySidecar container instead if that
 # is enabled.
-startupProbe:
+startupProbe: {}
```
Comment (corydolphin) [inline charts/daemonset-app/values.yaml:101]: In `simple-app` we set these to null. should we do that for consistency?

Comment (scohen-nd) [reply charts/daemonset-app/values.yaml]: Done

**Review by diranged — COMMENTED** [review]
Since this is a backwards incompatible change - can you bump the version to 1.14 and put in a README section in https://github.com/Nextdoor/k8s-charts/blob/main/charts/daemonset-app/README.md.gotmpl#L15-L28?

**Comment by scohen-nd** [comment]
> can you bump the version to 1.14 and put in a README section i

Done

**Comment by diranged** [comment]
@scohen-nd I don't see a readme update.. check the readme.md.gotmpl file for examples.

**Review by diranged — APPROVED** [review]

**Comment by scohen-nd** [comment]
> @scohen-nd I don't see a readme update.. check the readme.md.gotmpl file for examples.

Addressed that as well, my bad.

**Pull request merged**